src/sandstorm/supervisor.c++

```diff
@@ -663,9 +663,6 @@ kj::MainBuilder::Validity SupervisorMain::addCommandArg(kj::StringPtr arg) {
 // =====================================================================================
 
 kj::MainBuilder::Validity SupervisorMain::run() {
-  KJ_ASSERT(geteuid() != 0, "can't run as user ID zero (root)");
```

Comment:
Does this mean that if you set `SANDSTORM_USER=root` it'll now run the supervisor + grains as root?